core/adminserver/admin_server_test.go

```diff
@@ -53,9 +54,8 @@ type miniEnv struct {
 
 func newMiniEnv(ctx context.Context, t *testing.T) (*miniEnv, error) {
 	fakeDirectories := fake.NewDirectoryStorage()
-	if err := fakeDirectories.Write(ctx, &directory.Directory{
-		DirectoryID: "existingdirectory",
-	}); err != nil {
+	dir := &directory.Directory{DirectoryID: "existingdirectory"}
```

Comment:
You've detected a pattern of mine: passing arguments as an immediate.
Is there a more general pattern of not doing this that I should be aware of? 
If so, there's a lot of other code sites that could also use this change. (for other PRs)

Reply:
I prefer this because I find it generally more readable. My usual argument for this is..
If: You need to multi-line when constructing arguments in the function call.
Then: It must be the same number of lines to construct the input object before the function call.